go.mod

```diff
@@ -1,31 +1,31 @@
 module github.com/fullstorydev/grpcurl
 
-go 1.19
```

Comment:
That being said, our min supported version right now is actually go 1.21, so if we wanted to just update this directive to go 1.21 and then see what dependabot does from there, that would be acceptable